ran-core/src/test/java/io/ran/PropertyValueTest.java

```diff
@@ -0,0 +1,53 @@
+package io.ran;
+
+import io.ran.token.Token;
+import org.junit.Test;
+
+import java.lang.reflect.Constructor;
+import java.lang.reflect.InvocationTargetException;
+
+import static org.junit.Assert.*;
+
+public class PropertyValueTest {
+	private Property property = Property.get(Token.of("my"));
+
+	@Test
+	public void getNumericValue_simple_int() {
```

Comment:
Not sure if the name reflects the test here, another suggestion could be `propertyValue_simple_int_operations`

Reply:
4d8dbb7400a1150062e9a62104b9a716f85e8436